docs/api.md

```diff
@@ -840,6 +840,20 @@ logger.on('level-change', (lvl, val, prevLvl, prevVal) => {
 logger.level = 'trace' // trigger event
 ```
 
+Please note that due to a [known bug](https://github.com/pinojs/pino/issues/1006), every `logger.child()` call will
+fire a `level-change` event, with `levelLabel` equal to `previousLevelLabel` and `levelValue` equal to `previousLevelValue`.
+You might want to ignore these events, using something like:
+
+```js
+const logger = require('pino')()
+logger.on('level-change', (lvl, val, prevLvl, prevVal) => {
```

Comment:
I would remove the arrow function and rely on `this === logger` to check.

Reply:
Thank you, I did not think of this (no pun intended) :)